probe-rs/targets/MSPM0L_Series.yaml

```diff
@@ -124,7 +124,6 @@ variants:
     - main
   flash_algorithms:
   - mspm0l_main_16kb
-  - mspm0l_nonmain
```

Comment:
Where does this change come from?

Reply:
These chips don't have enough memory to hold these flash algos.